Cargo.toml

```diff
@@ -15,7 +15,7 @@ wasmer-compiler = { version = "1.0.0-beta1", path = "lib/compiler" }
 wasmer-compiler-cranelift = { version = "1.0.0-beta1", path = "lib/compiler-cranelift", optional = true }
 wasmer-compiler-singlepass = { version = "1.0.0-beta1", path = "lib/compiler-singlepass", optional = true }
 wasmer-compiler-llvm = { version = "1.0.0-beta1", path = "lib/compiler-llvm", optional = true }
-wasmer-emscripten = { version = "1.0.0-beta1", path = "lib/emscripten", optional = true }
+#wasmer-emscripten = { version = "1.0.0-beta1", path = "lib/emscripten", optional = true }
```

Comment:
Not that I think you've forgotten, but since this isn't a draft PR, I'll note that this PR can't land with this change.